doc/architecture/software-update.rst

```diff
@@ -48,22 +77,55 @@ and applying the updates.
 It can also restore modified files to their pristine state as then were in the
 initial sw image.
 
+.. note::
+The version of swupd client that ships with Ostro 1.0 does not support yet
+signature verification, therefore it is necessary to provide the swupd feed
+through a secure channel (https).
+Once this is addressed, also unsecure channels will be suitable, because,
+as long as the signature and integrity checks are satisfied,
+the software update client will treat them as trusted.
 
 SW Bundles
 ==========
 
 The software update mechanism supports the concept of bundles_: groups of files
 that provide one functionality.
-It is similar to the concept of a package used in the vast majority of Linux distros,
-but in general one software update bundle maps to several packages (ex: .rpm or .deb)
 
-There is one main os-core bundle containing everything that is mandatory for having
-a bare-bones working system.
+Ostro users coming from popular Linux distros are probably already familiar with
+the concept of package.
+How is a bundle different from a pacakge?
+
+- A bundle doesn't have any dependency, it is self contained.
```

Comment:
A bundle has no dependencies: it is self-contained. 
A bundle might also list files contained in other bundles, and that is permitted by design.
The software update server and client tools work together to properly handle any such overlapping.